OpenEXR/IlmImfTest/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ add_executable(IlmImfTest
   testXdr.cpp
   testYca.cpp
   testLargeDataWindowOffsets.cpp
+  random
```

Comment:
random.cpp ?

Reply:
Fixed.  Interesting that it seems to have worked, but better to be consistent.